spec/lib/capsule_crm/serializer_spec.rb

```diff
@@ -130,5 +131,16 @@ class SerializableInverse
           to eql(Date.today.to_s(:db))
       end
     end
+
+    context 'when there are datetimes' do
+      let(:time) { Time.now }
+
+      before { object.a_timestamp = time }
+
+      it 'uses the UTC time' do
+        expect(subject['serializabletest']['aTimestamp'])
+          .to eql(time.utc.strftime("%Y-%m-%dT%H:%M:%SZ"))
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.